frontend/src/routes/(frontend)/[accountId]/[statusId]/index.tsx

```diff
@@ -4,20 +4,25 @@ import Status from '~/components/Status'
 import { formatDateTime } from '~/utils/dateTime'
 import { formatRoundedNumber } from '~/utils/numbers'
 import * as statusAPI from 'wildebeest/functions/api/v1/statuses/[id]'
+import * as contextAPI from 'wildebeest/functions/api/v1/statuses/[id]/context'
 import { Link, loader$ } from '@builder.io/qwik-city'
 import StickyHeader from '~/components/StickyHeader/StickyHeader'
 import { Avatar } from '~/components/avatar'
 
 export const statusLoader = loader$<
 	{ DATABASE: D1Database; domain: string },
 	Promise<{ status: MastodonStatus; context: StatusContext }>
->(async ({ redirect, platform, params }) => {
-	const response = await statusAPI.handleRequest(platform.DATABASE, params.statusId)
-	const results = await response.text()
-	if (!results) {
+>(async ({ request, redirect, platform, params }) => {
+	const domain = new URL(request.url).hostname
+	const statusResponse = await statusAPI.handleRequest(platform.DATABASE, params.statusId, domain)
+	const statusText = await statusResponse.text()
+	if (!statusText) {
 		throw redirect(303, '/not-found')
 	}
-	return { status: JSON.parse(results), context: { ancestors: [], descendants: [] } }
+	const contextResponse = await contextAPI.handleRequest(domain, platform.DATABASE, params.statusId)
+	const contextText = await contextResponse.text()
```

Comment:
```suggestion
	const contextText = await contextResponse.json()
```
can't you parse a JSON directly?

Reply:
Sadly not. The `resp.json()` function returns a weird object that Qwik is not able to serialize.

Reply:
I don't like that Qwik has those implicit things that don't work (same with state at the root or something like that) but ok.

Reply:
yeah, unfortunately Qwik does create these sort of constraints 😓 

(although I think it is often understandable given the core idea of Qwik of splitting things up as much as possible)